man/systemd.network.xml

```diff
@@ -809,6 +809,14 @@
             <para>As in the <literal>[Network]</literal> section.</para>
           </listitem>
         </varlistentry>
+         <varlistentry>
+           <term><varname>GatewayOnlink=</varname></term>
+           <listitem>
+             <para>Explicitly consider the gateway to be onlink, even if there
```

Comment:
"onlink" isn't an english word. Can you elaborate on this here, and what this means?